test/python/circuit/test_circuit_operations.py

```diff
@@ -411,17 +411,22 @@ def test_copy_empty_like_circuit(self):
         copied = qc.copy_empty_like("copy")
         self.assertEqual(copied.name, "copy")
 
+    # pylint: disable=invalid-name
```

Comment:
I think this comment was supposed to go inside the function so it scopes to only that, but I also don't care because I don't like this rule haha.